.github/workflows/deploy.yml

```diff
@@ -26,4 +26,5 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           publish_dir: ./.build
           publish_branch: vh_page
+          force_orphan: true
```

Comment:
This should be `false`. `true` is the default value, and this forces a single commit in the `publish_branch`. We do not want single commit, so that each commit diff is meaningful, so it should be `false`.